### DISCUSSION TITLE
Best way to control manifest folder when deploying multiple control nodes

### DISCUSSION THREAD

**Discussion by ssharma2089** [k3s-io/k3s#5368] [General]
We are worried about deploying multiple manifest folders in the control nodes. Is there any recommended way to control for potential conflict? 
Can we disable manifest folder on certain nodes?

**Comment by ssharma2089** [comment]
We have found the below jira. Is it recommended way?
https://github.com/k3s-io/k3s/issues/1445

Above flag in the issue is not mentioned in k3s documentation.

**Reply by brandond** [reply]
The embedded helm controller has nothing to do with the deploy controller; they serve different functions.

K3s does not make any attempt to ensure that the contents of the manifests directory are synchronized across nodes. Files in that directory are applied on node startup, and whenever the timestamp and contents change while K3s is running.

Best practice would probably be to either pick a single primary node to deploy your manifests from, or if you want to place the files on multiple nodes, use popular system administration tools (ansible, puppet, rsync, etc) to keep the files in sync across all the nodes.
